operator/BACKPORTING.md

```diff
@@ -0,0 +1,76 @@
+# Backporting upstream PRs into an OpenShift Logging Release Branch
+
+The following instructions describe how to backport a merged Loki-Operator upstream PR into an OpenShift Logging release branch.
+
+## Background
+
+OpenShift Logging releases branches for the Loki Operator follow the product version scheme. In detail each minor version of the OpenShift Logging 5 release streams yield individual `release-5.y` release branches. For example, OpenShift Logging `5.5.0 - 5.5.z` expects that the bundled operators are build from a corresponding branch named `release-5.5`.
+
+## Instructions
+
+The following approach describes how to backport an upstream PR merged into the [grafana/loki][grafana-loki] `main` branch.
+
+__Note:__ The grafana/loki upstream repository follows a squash-and-merge approach when merging PRs. This means each PR represents one commit in the `main` branch history. In case you had to create follow-up PRs for the same issue because of flaws or incompleteness identified after merge, you are encouraged to combine them in a **single** backport PR.
+
+### Ensure you have this repository as a local remote
+
+```shell
+$ git remote add openshift git@github.com:openshift/loki.git
+```
+
+### Identify the PR to backport
+
+Use the upstream [changelog][loki-operator-changelog] file to identify the PR you need to backport, e.g. [grafana/loki#7000](https://github.com/grafana/loki/pull/7000).
+
+### Create a local backport branch
+
+```shell
+$ git co -b backport-pr-7000 openshift/release-5.5
+```
+
+### Cherry pick your PR into the backport branch
+
+Open the backport candidate PR [grafana/loki#7000](https://github.com/grafana/loki/pull/7000) and scroll to bottom to identify the merge commit into `main`, e.g.:
+
+```
+periklis merged commit 766d4b7 into grafana:main 20 days ago
+```
+
+and cherry-pick:
+
+```shell
+$ git cherry-pick 766d4b7
+```
+
+__Note:__ When backporting PRs from main merge conflicts can arise because our upstream `main` history includes developed features and refactorings for the next minor release.
```

Comment:
```suggestion
__Note:__ When backporting PRs from `main`, merge conflicts can arise because our upstream `main` history includes developed features and refactorings for the next minor release.
```